ohapi/public.py

```diff
@@ -98,6 +99,7 @@ def download(source, username, directory, max_size, verbose, debug):
         logging.basicConfig(level=logging.DEBUG)
     elif verbose:
         logging.basicConfig(level=logging.INFO)
+    logging.basicConfig(level=logging.DEBUG)
```

Comment:
This is overriding the flags for `--debug` and `--verbose` that are used to set logging level. (i.e. the four lines preceding thish one.) Instead, it's forcing *all* usage of this function to produce DEBUG logging data. Why? Were you unaware of (or unable to use) the `--debug` flag to accomplish this?

Reply:
`--debug` flag would print all the logging statements which are a lot of logging statements as is this PR doing currently. I understand that this should not be the appropriate behavior and wanted to discuss exactly which statements should be printed to give a minimal feedback to the user about what exactly is going on when he runs a given command. Currently not even a single message is shown in terminal which leaves the user in suspicion whether the files are being downloaded or not. and when we `ls` it through another terminal we see that multiple copies have been downloaded.